src/core/sync/config.d

```diff
@@ -28,7 +28,7 @@ version (Posix)
     {
         static if ( is (typeof ( pthread_condattr_setclock ) ) )
         {
-            clock_gettime( CLOCK_MONOTONIC, &t );
+            clock_gettime( CLOCK_REALTIME, &t );
```

Comment:
Can you add a comment so that it doesn't get changed back?

Reply:
I added the comment but cant think of a test case.

Reply:
OK neither can I.